pkg/util/log/log.go

```diff
@@ -120,7 +119,7 @@ func (sw *DatadogLogger) changeLogLevel(level string) error {
 	if !ok {
 		return errors.New("bad log level")
 	}
-	Logger.level = lvl
+	logger.Load().level = lvl
```

Comment:
Do you need to check nil here?

Reply:
The change of the type of pointer of the `logger` variable didn’t change the execution flow.
So, if checking `nil` wasn’t needed before, it shouldn’t be needed now either.
The code must probably already guarantee that `changeLogLevel` cannot be called before `SetupLogger`.